testbed/windows/win32_window.cc

```diff
@@ -13,14 +13,24 @@ namespace {
 // constant for machines running at 100% scaling.
 constexpr int kBaseDpi = 96;
 
-constexpr LPCWSTR kClassName = L"CLASSNAME";
+constexpr const wchar_t kClassName[] = L"CLASSNAME";
 
 // Scale helper to convert logical scaler values to physical using passed in
 // scale factor
 int Scale(int source, double scale_factor) {
   return static_cast<int>(source * scale_factor);
 }
 
+// Returns the DPI for the monitor containing, or closest to, |point|.
+UINT GetDpiForMonitorAtPoint(const Win32Window::Point &point) {
+  const POINT target_point = {static_cast<LONG>(point.x),
+                              static_cast<LONG>(point.y)};
+  HMONITOR monitor = MonitorFromPoint(target_point, MONITOR_DEFAULTTONEAREST);
```

Comment:
The reason for this change is that when I was fixing warnings I discovered that passing a null `HWND` to `MonitorFromWindow` is actually a violation of the annotations on the method. The recommended way to get the primary monitor is to call this with a 0,0 point rather than using MonitorFromWindow with null. We should fix that in the embedding.